tests/utilities/test_planning_handler.py

```diff
@@ -1,10 +1,10 @@
 from unittest.mock import patch
```

Comment:
In this file, It would be nice if we added a few more tests to test the initialization of CrewPlanner is working properly.

- one test that passes in an agent 
- One test that doesn't pass in an agent


![Screenshot 2024-07-25 at 11 38 49 AM](https://github.com/user-attachments/assets/4f603fa9-918f-4692-9f09-f834b66c4f67)